### ISSUE TITLE
Softmax Cross-Entropy Loss Function

### ISSUE THREAD

**Issue by prijall** [Open-Deep-ML/DML-OpenProblem#452]
It’s about stable softmax which is just a numerically safer way to compute the softmax function which is used in cross-entropy loss computation and  can be beneficial for beginner!!!

**Comment by moe18** [comment]
sounds good

**Comment by prijall** [comment]
what will be my problem number??

**Comment by moe18** [comment]
finish the question and I will assign it at the end, easier to keep track of questions this way
